lib/bloc/account/account_bloc.dart

```diff
@@ -167,11 +163,11 @@ class AccountBloc extends Cubit<AccountState> with HydratedMixin {
   }
 
   Future<sdk.LnUrlCallbackStatus> lnurlAuth({
-    required sdk.LnUrlAuthRequestData reqData,
+    required sdk.LnUrlAuthRequestData data,
   }) async {
-    _log.info("lnurlAuth amount: reqData: $reqData");
+    _log.info("lnurlAuth amount: data: $data");
     try {
-      return await _breezLib.lnurlAuth(reqData: reqData);
+      return await _breezLib.lnurlAuth(reqData: data);
```

Comment:
This is an exception as there's no wrapping LnUrlAuthRequest struct for LnUrlAuthRequestData. It can stay as reqData.

Reply:
Reverted